### PR TITLE
Add Listen Network prefix

### PR DESCRIPTION
**Pull request by Irish-bee**
We have now completed the cross-chain and will be auctioning the kusana slots in the next two weeks or so. Whether we can pass this PR as soon as possible? we need to carry out the last round of testing. thank you!

### CODE REVIEW

**Comment by Irish-bee** [comment]
@bkchr hello man, Whether Kusama intentionally controlled the number of technical teams that participated in the slot auction at the same time?

**Comment by Irish-bee** [comment]
> @bkchr hello man, Whether Kusama intentionally controlled the number of technical teams that participated in the slot auction at the same time?

@BenWhiteJam are you here?

**Comment by bkchr** [comment]
> @bkchr hello man, Whether Kusama intentionally controlled the number of technical teams that participated in the slot auction at the same time?

How is the ss58 prefix related to Kusama slot auctions?

**Comment by Irish-bee** [comment]
> > @bkchr hello man, Whether Kusama intentionally controlled the number of technical teams that participated in the slot auction at the same time?
> 
> How is the ss58 prefix related to Kusama slot auctions?

Do you mean it is not necessary for each parallel chain to have its own SS58 prefix, or should these parallel chains set their own SS58 after bidding for slots and starting the main network?

**Comment by bkchr** [comment]
For registering a wasm blob on Kusama, you don't need any ss58. Ss58 also never hits the runtime, so we are not blocking here anything.

**Comment by Irish-bee** [comment]
> For registering a wasm blob on Kusama, you don't need any ss58. Ss58 also never hits the runtime, so we are not blocking here anything.

So, parallel chains will actually use the default 42 and not have a negative impact on any cross-chain operations, right?

**Comment by Irish-bee** [comment]
> > For registering a wasm blob on Kusama, you don't need any ss58. Ss58 also never hits the runtime, so we are not blocking here anything.
> 
> So, parallel chains will actually use the default 42 and not have a negative impact on any cross-chain operations, right?

@BenWhiteJam hello man, How long will it take to merge this PR? We really need this SS58 Prefix.

**Comment by BenWhiteJam** [comment]
Can you please reach out on Element https://matrix.to/#/@benwhitejam:matrix.parity.io 
Other than that, as Basti said, if you are in a hurry, go ahead and setup your Parathread/Crowdloan.

**Comment by DanielCake-Baly** [comment]
> 

@Irish-bee  hey, I'm looking into this request, could you please reach out to me on [Element](https://matrix.to/#/@daniel.cake-baly:matrix.parity.io) ?

**Review by BenWhiteJam — APPROVED** [review]

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (bkchr) [inline .idea/.gitignore:1]: Please remove the file.

**Review by bkchr — APPROVED** [review]
Please remove the file and merge master

**Comment by DanielCake-Baly** [comment]
> Please remove the file and merge master

Hey @Irish-bee, are you here?

**Comment by itsbirdo** [comment]
Closing this as Listen network is no longer building on Kusama.

**Pull request closed without merging**